togetherly/lib/utilities/date.dart

```diff
@@ -0,0 +1,14 @@
+class DateHelpers {
+  DateTime getDateToday() {
+    DateTime date = DateTime.now();
+    return DateTime(date.year, date.month, date.day);
+  }
+
+  bool isDueToday(DateTime dueDate) {
+    return dueDate.toString().split(' ')[0] == getDateToday().toString();
```

Comment:
I still think that doing this without strings will be more readable and robust. At the moment, the logic here simply doesn't work, as is evidenced by running the following code in DartPad:
```dart

void main() {
  DateTime now = DateTime.now();
  DateTime today = DateTime(now.year, now.month, now.day);
  DateTime dueDate = today;
  
  print(dueDate.toString().split(' ')[0]);
  print(today.toString());
  
  print(dueDate.toString().split(' ')[0] == today.toString());
}
```
It's not a big deal though, so I'll push some changes that make this more concise, readable, and robust.